spec/javascript/packages/document-capture/components/review-issues-step-spec.jsx

```diff
@@ -68,6 +68,41 @@ describe('document-capture/components/review-issues-step', () => {
       }),
     ).to.exist();
   });
+  it('renders initially with warning page and displays attempts remaining with IPP', () => {
```

Comment:
This test passes on `main`, which doesn't give much assurance that it protects against regressions. I think the issue is with the _non_-IPP flow? Can we carry over the enhancements to existing spec for the non-IPP flow?

Reply:
Thanks for porting over those improvements. Thanks also for adding the missing coverage for IPP flows.